packages/app/src/app/components/Preview/DevTools/Terminal/index.tsx

```diff
@@ -22,7 +22,7 @@ type State = {
 // Incredibly hacky way of letting the StatusBar access the state of the component.
 // In the future we need to abstract this away to the global state and dispatch an event.
 // We need to keep all the tabs in the global state and work from there.
-let createShell;
+let createShell: () => void | undefined;
```

Comment:
@CompuIves If a variable can  be undefined you can just declare it with `?:`

```suggestion
let createShell?: () => void;
```